calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -88,9 +92,41 @@ spec:
   webApplicationFirewall: Enabled
 ```
 
+#### Disabling WAF
+
+To disable WAF update the [ApplicationLayer](https://docs.tigera.io/reference/installation/api#operator.tigera.io/v1.ApplicationLayer) resource to include the `webApplicationFirewall` field, and ensure it is set to `Disabled`.
+
+Example:
+
+```yaml
+apiVersion: operator.tigera.io/v1
+kind: ApplicationLayer
+metadata:
+  name: tigera-secure
+spec:
+  webApplicationFirewall: Disabled
+```
+
 #### Step 2: Select traffic for WAF
 
-Annotate the services you wish to enable WAF for as shown.
+Once your cluster is configured for WAF you will be presented with a list of Kubernetes services available in the cluster:
+
+<img
+  src='/img/calico-enterprise/master/waf-services.png'
+  alt='WAF services'
+  width='600'
+/>
+
+From the **Action** column you can then select **Enable** or **Disable** action to respectively enable or disable WAF for each of the services.
+You will be presented with a window containing a **kubectl** command that you will be required to execute for the action to take effect.
```

Comment:
Suggestion: You will be presented with a window containing a kubectl command, which you must execute on a cluster node for the action to take effect.
Note: kubectl does not need to be in bold faces.